framework/source/class/qx/lang/normalize/Object.js

```diff
@@ -58,6 +58,27 @@ qx.Bootstrap.define("qx.lang.normalize.Object", {
       }
 
       return arr;
+    },
+
+
+    /**
+     * Determines whether two values are the same value.
+     *
+     * <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is">MDN web docs: Object.is()</a>
+     *
+     * @signature function(value1,value2)
+     * @param value1 {Object} the first value to compare
+     * @param value2 {Object} the second value to compare
+     * @return {Boolean} indicating whether or not the two arguments are the same value.
+     */
+    is : function(v1, v2) {
+      if (v1 === 0 && v2 === 0) {
+        return 1 / v1 === 1 / v2;
+      }
+      if (v1 !== v1) {
```

Comment:
This comparison always yields false, doesn't it? If there's some strange edge case where that's not true, it certainly needs documentation here to specify what it is.

Reply:
@derrell this is not my code. It is the hopfully well tested code recommende by MDN. See:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is

Reply:
Disregard that comment. NaN !== NaN.

However, see next comment.